editor/components/page-attributes/test/check.js

```diff
@@ -39,8 +39,20 @@ describe( 'PageAttributesCheck', () => {
 		expect( wrapper.type() ).toBe( null );
 	} );
 
-	it( 'should render if page attributes support', () => {
-		const wrapper = shallow( <PageAttributesCheck postType={ postType }>content</PageAttributesCheck> );
+	it( 'should render if page attributes support is true and no available templates exist', () => {
+		const wrapper = shallow( <PageAttributesCheck availableTemplates={ {} } postType={ postType }>content</PageAttributesCheck> );
+
+		expect( wrapper.type() ).not.toBe( null );
```

Comment:
I'm wondering if having test like:

```js
expect( wrapper.find( 'PageAttributesCheck' ) ).toHaveText( 'content' );
// not sure it works though :)
```
would better document the behavior of the component.